src/relay/backend/contrib/tensorrt/enable_tensorrt.cc

```diff
@@ -0,0 +1,569 @@
+                  << " not supported: cannot modify batch dimension.";
+        return false;
+      }
+    }
+  }
+  return true;
+}
+
+bool ConcatenateOpChecker(const CallNode* call, const std::string& op_name,
+                          const std::tuple<int, int, int>& trt_version) {
+  const auto* concat_attr = call->attrs.as<ConcatenateAttrs>();
+  if (concat_attr->axis == 0) {
+    LOG(INFO) << op_name << " not supported: cannot modify batch dimension.";
+    return false;
+  }
+  return true;
+}
+
+bool BiasAddOpChecker(const CallNode* call, const std::string& op_name,
+                      const std::tuple<int, int, int>& trt_version) {
+  auto shape0 = GetShape(call->type_args[0]);
+  if (shape0.size() < 2 || shape0.size() > 4) {
+    LOG(INFO) << op_name << " not supported: input must be rank 2, 3, or 4.";
+    return false;
+  }
+  return true;
+}
+
+bool Conv2DTransposeOpChecker(const CallNode* call, const std::string& op_name,
+                              const std::tuple<int, int, int>& trt_version) {
+  const auto* conv2d_attr = call->attrs.as<Conv2DTransposeAttrs>();
+  if (conv2d_attr->data_layout != "NCHW" ||
+      conv2d_attr->kernel_layout != "OIHW") {
+    LOG(INFO) << op_name << " not supported: must be NCHW.";
+    return false;
+  }
+  if (conv2d_attr->out_layout != "" && conv2d_attr->out_layout != "NCHW") {
+    LOG(INFO) << op_name << " not supported: must be NCHW.";
+    return false;
+  }
+  if (conv2d_attr->dilation[0].as<IntImm>()->value != 1 ||
+      conv2d_attr->dilation[1].as<IntImm>()->value != 1) {
+    LOG(INFO) << op_name << " not supported: dilation rate must be 1.";
+    return false;
+  }
+  return true;
+}
+
+bool TransposeOpChecker(const CallNode* call, const std::string& op_name,
+                        const std::tuple<int, int, int>& trt_version) {
+  const auto* attrs = call->attrs.as<TransposeAttrs>();
+  if (attrs->axes[0].as<IntImm>()->value != 0) {
+    LOG(INFO) << op_name << " not supported: can't modify batch dimension.";
+    return false;
+  }
+  return true;
+}
+
+bool ReshapeOpChecker(const CallNode* call, const std::string& op_name,
+                      const std::tuple<int, int, int>& trt_version) {
+  const auto* attrs = call->attrs.as<ReshapeAttrs>();
+  // TODO(trevmorr): check for modified batch dim.
+  for (size_t i = 0; i < attrs->newshape.size(); i++) {
+    if (attrs->newshape[i].as<IntImm>()->value < -1) {
+      LOG(INFO) << op_name << " not supported: reshape dims must be explicit.";
+      return false;
+    }
+  }
+  return true;
+}
+
+bool PadOpChecker(const CallNode* call, const std::string& op_name,
+                  const std::tuple<int, int, int>& trt_version) {
+  const auto* attrs = call->attrs.as<PadAttrs>();
+  if (attrs->pad_mode != "constant") {
+    LOG(INFO) << op_name << " not supported: pad mode must be constant.";
+    return false;
+  } else if (attrs->pad_value != 0.0) {
+    LOG(INFO) << op_name << " not supported: pad value must be zero.";
```

Comment:
I both of the messages above I'd output current `pad_mode` and `pad_value`.
```
LOG(INFO) << op_name << " not supported: pad mode is %s but must be constant.";

LOG(INFO) << op_name << " not supported: pad value is %f but must be 0.0."
```

Reply:
Thanks, I went ahead and included current values for the rest of the ops messages as well.